plugins/catalog/src/state/filters.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity } from '@backstage/catalog-model';
+import { isOwnerOf, isStarredEntity } from '@backstage/plugin-catalog-react';
+import { EntityFilter, EntityFieldFilter } from './types';
+import { get } from 'lodash';
+
+type StateProps = {
```

Comment:
If I understand things right, maybe this could be given a more full name such as `EntityPredicateEnvironment` or something along those lines? It's only meant to be passed as contextual info to the predicates isn't it